api/wasm/indigo-ketcher/indigo-ketcher.cpp

```diff
@@ -333,7 +389,7 @@ namespace indigo
         }
     }
 
-    void fmtDouble2VecStr(double arg, std::vector<std::string>& vec)
+    void pushDoubleAsStr(double arg, std::vector<std::string>& vec)
```

Comment:
(This is about line 403 - cannot attach comment to old unchanged line):
```
if (str.size())
```
I doubt that we should anyway skip adding the element to the vector. 
Once the function is called, the corresponding "cell" needs to be added even with empty string.